corehq/apps/domain/management/commands/delete_domain.py

```diff
@@ -40,6 +41,11 @@ def handle(self, domain_name, **options):
             if confirm != domain_name:
                 print("\n\t\tDomain deletion cancelled.")
                 return
+
         print("Deleting domain {}".format(domain_name))
         domain.delete()
+
+        for hqESQuery in [AppES, CaseES, CaseSearchES, DomainES, FormES, GroupES, LedgerES, UserES]:
+            assert hqESQuery().domain(domain_name).run().total == 0, '%s contains data' % hqESQuery.index
```

Comment:
Might be nice to extend this to explain what the user needs to do, even just "that should be manually deleted".

Reply:
Why not just delete it automatically in this script?